app/models/suggested_edit.rb

```diff
@@ -10,6 +10,12 @@ class SuggestedEdit < ApplicationRecord
   has_and_belongs_to_many :tags
   has_and_belongs_to_many :before_tags, class_name: 'Tag', join_table: 'suggested_edits_before_tags'
 
+  after_save :clear_pending_cache
+
+  def clear_pending_cache
+    Rails.cache.delete "pending_suggestions/#{post.category_id}"
```

Comment:
query: At the moment, this clears the indicator after _any_ save of _any_ suggested edit in the category. Is that intended, or do we need to add some logic here to only clear it if it's actually clear?

Reply:
It was just the simplest way I could think of to trigger a recalculation of whether there are any pending suggestions; it probably could be made a bit more efficient